docs/features/software-templates/ui-options-examples.md

```diff
@@ -441,3 +441,30 @@ repoUrl:
 `secretsKey` is the key used within the template secrets context to store the credential and `additionalScopes` is any additional permission scopes to request.
 
 The supported `additionalScopes` values are `gerrit`, `github`, `gitlab`, `bitbucket`, and `azure`.
+
+## RepoBranchPicker
```

Comment:
I'm not sure we should document this at this point if it's Bitbucket only as per the PR description or if we do this should be noted and a clear plan to add support for others should be outlined and linked to an issue where Adopters can see the progress

Reply:
Good point. Maybe that's something that should be discussed during the scaffolder SIG.